hw2/03-count.html

```diff
@@ -9,25 +9,27 @@
       href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
     <script defer src="03-count.js"></script>
   </head>
-  <body>
+  <body class="bg-primary">
```

Comment:
Regarding count.html:
The background color does not match the screenshot. I had trouble finding a bootstrap solution to this myself and opted to use the style attribute (style="background-color: somehexval") as a potential solution.